roles/satellite-clone/tasks/main.yml

```diff
@@ -135,10 +135,11 @@
     state: latest
 
 # due to support for remote databases in 6.4 the postgres user (and server) may be missing
-- name: Install postgresql-server
-  yum:
-    name: "postgresql-server"
-    state: latest
+# Removing this as 6.8 will have rh-postgresql12-postgresql-server
+#- name: Install postgresql-server
+#  yum:
+#    name: "postgresql-server"
+#    state: latest
```

Comment:
We need to keep things backwards-compatible since the latest sat-clone version is shipped to all Satellite versions.

With ansible, this step won't perform any action if postgresql-server is found. This should stay as a step unless there is something I'm missing.

Reply:
@johnpmitsch  Thank you for reviewing.

To me it looks like we need to add a condition, if 6.8 then install rh-postgresql12-postgresql-server else postgresql-server. I am learning ansible but I will try modifying this and re-send a PR

Reply:
we have been just specifying a range manually:

```yaml
  do_something:
     when: satellite_version in ["6.2", "6.3", "6.4", "6.5", "6.6", "6.7"]

  do_other_thing:
    when: satellite_version in ["6.8"]
```

There may be a better way to do it, but version comparisons can be tricky, so we've just gone with this. Feel free to keep this convention or suggest anything you've come up with!

Reply:
@johnpmitsch I have added the change accordingly